plugins/import/import.ts

```diff
@@ -14,7 +13,7 @@ export async function loadContract(source: string, network: string, address: str
   }
   switch (source) {
     case 'etherscan':
-      return await loadEtherscanContractMemoized(network, address);
+      return await loadEtherscanContract(network, address);
```

Comment:
i think we're going to miss memoizing this, no?

Reply:
I don't ever see it being a hit, and probably because we already have multiple levels of caching

Reply:
👌